src/Grpc/Grpc.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <PackageDescription>The gRPC Protobuf .NET services for Durable Task Framework.</PackageDescription>
+  </PropertyGroup>
```

Comment:
What's the name of this nuget package? I assume it's hidden away via some dynamic MSBuild configuration?

Reply:
It is `Microsoft.DurableTask.Grpc`

Reply:
I'm good with `Microsoft.DurableTask.Grpc` since we have gRPC-specific dependencies we're pulling in.